### DISCUSSION TITLE
How to publish by custom domain

### DISCUSSION THREAD

**Discussion by phibersoft** [vercel/vercel#4312] [General] [answered]
I'm pretty new at NextJS. I'm trying to learn the deployment phase and got stuck somewhere.

I proceeded according to the documents. I changed the domain (by Godaddy) dns settings. Currently, my domain appears to be approved. Everything is green, there is no problem. But when I go to the domain, I still can't find anything. Many thanks in advance for the help.

![Screenshot_2020-05-08 Project Settings - Vercel](https://user-images.githubusercontent.com/54160817/81444284-8f57ee00-917f-11ea-8b52-65d593935733.png)

**Comment by coetry** [comment, marked as answer]
DNS is tricky and sometimes takes a bit of time to propagate everywhere, but I was able to access your site fine :smile: 

![image](https://user-images.githubusercontent.com/10926503/81451479-37eb5b00-914a-11ea-9902-87fd2fdc9f4d.png)

```sh
▲ ~  curl -I https://specdown.com
HTTP/2 200 
date: Fri, 08 May 2020 21:38:15 GMT
content-type: text/html
cache-control: public, max-age=0, must-revalidate
content-length: 20053
content-disposition: inline
access-control-allow-origin: *
accept-ranges: bytes
etag: W/"cd23195748b52d280062ae5c5bc73d5ce305add23567737786220cfb1a8a8398"
x-vercel-cache: PRERENDER
x-now-cache: PRERENDER
age: 0
x-now-trace: oma1
server: now
x-vercel-id: oma1::c4rp9-1588973895417-b0aab4d3fde1
x-now-id: oma1::c4rp9-1588973895417-b0aab4d3fde1
strict-transport-security: max-age=63072000
```

**Reply by coetry** [reply]
For others who come across this later, nothing more is required for custom domains apart from the instructions [here](https://vercel.com/docs/v2/custom-domains).

**Reply by phibersoft** [reply]
I apologize for being busy. I wasn't expecting such a result 😄 

Regards..
